### DISCUSSION TITLE
jellyseerr/sonarr metadata source mismatch

### DISCUSSION THREAD

**Discussion by jamilnielsen** [seerr-team/seerr#849] [Q&A] [answered]
it appears that jellyseerr relies on metadata that by default is different from what sonarr uses lets use Bleach as an example.
Jellyseerr: 2 seasons.
sonarr: 17 seasons.
result: jellyseer tells sonarr to import the first 2 seasons.
do we have a fix for this or do we need to make an issue.

**Comment by fallenbagel** [comment]
This is nothing we can do. This is not a bug. We use tmdb. sonarr uses tvdb. Both databases refuses to stick to a standard.

**Reply by jamilnielsen** [reply]
im assuming its not within plans to support both tmdb and tvdb?

**Reply by fallenbagel** [reply, marked as answer]
> im assuming its not within plans to support both tmdb and tvdb?

Tvdb api is closed source and paid.
